libs/openchallenges/ui/src/lib/navbar/navbar.component.html

```diff
@@ -1,8 +1,7 @@
 <!-- TODO: figure out if the <nav> should go inside of a <header> element. -->
 <nav class="sage-navbar-header" aria-label="Top Toolbar">
   <a mat-button class="home-btn" routerLink="/" aria-label="Sage Angular">
-    <img class="logo" src="assets/images/registry.png" alt="OpenChallenges" />
-    <span>{{ title }}</span>
+    <img class="logo" src="assets/images/oc-white.svg" alt="OpenChallenges" />
```

Comment:
```suggestion
    <img class="logo" src="assets/images/oc-white.svg" alt="OpenChallenges logo" />
```

Reply:
I opted for just "OpenChallenges" here, since that's what we're trying to display (that is, the logo + app name).  

In `about.component.html`, we are indeed just displaying the logo icon.  To make it more clear, I will use "icon" instead of "logo" in about's alt text.

Reply:
That makes sense, thanks!